tests/test_clos/test_ssl_selector.py

```diff
@@ -0,0 +1,43 @@
+"""Pytest added to test ssl_selector function"""
+import contextlib
+import os
+import sys
+from io import StringIO
+from cloudos.__main__ import ssl_selector
```

Comment:
```suggestion
from cloudos.__main__ import ssl_selector
import warnings
```
This will be required to check if warnings were really disabled.